spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -95,363 +131,387 @@
-      let(:residential_address_proof) do
-        instance_double(Proofing::Resolution::Result)
-      end
-      let(:resolution_result) do
-        instance_double(Proofing::Resolution::Result)
-      end
-      let(:ipp_enrollment_in_progress) { true }
-      let(:double_address_verification) { true }
-      let(:applicant_pii) do
-        JSON.parse(<<-STR, symbolize_names: true)
-            {
-              "uuid": "3e8db152-4d35-4207-b828-3eee8c52c50f",
-              "middle_name": "",
-              "phone": "",
-              "state_id_expiration": "2029-01-01",
-              "state_id_issued": "MI",
-              "first_name": "Imaginary",
-              "last_name": "Person",
-              "dob": "1999-09-00",
-              "identity_doc_address1": "1 Seaview",
-              "identity_doc_address2": "",
-              "identity_doc_city": "Sant Cruz",
-              "identity_doc_zipcode": "91000",
-              "state_id_jurisdiction": "AZ",
-              "identity_doc_address_state": "CA",
-              "state_id_number": "AZ333222111",
-              "same_address_as_id": "false",
-              "state": "MI",
-              "zipcode": "48880",
-              "city": "Pontiac",
-              "address1": "1 Mobile Dr",
-              "address2": "",
-              "ssn": "900-32-1898",
-              "state_id_type": "drivers_license",
-              "uuid_prefix": null
-            }
-        STR
-      end
-      let(:residential_address) do
-        {
-          address1: applicant_pii[:address1],
-          address2: applicant_pii[:address2],
-          city: applicant_pii[:city],
-          state: applicant_pii[:state],
-          state_id_jurisdiction: applicant_pii[:state_id_jurisdiction],
-          zipcode: applicant_pii[:zipcode],
-        }
-      end
-      let(:state_id_address) do
-        {
-          address1: applicant_pii[:identity_doc_address1],
-          address2: applicant_pii[:identity_doc_address2],
-          city: applicant_pii[:identity_doc_city],
-          state: applicant_pii[:identity_doc_address_state],
-          state_id_jurisdiction: applicant_pii[:state_id_jurisdiction],
-          zipcode: applicant_pii[:identity_doc_zipcode],
-        }
-      end
-
-      context 'LexisNexis InstantVerify passes for residential address' do
-        before do
-          allow(instance).to receive(:resolution_proofer).and_return(instant_verify_proofer)
-          allow(instant_verify_proofer).to receive(:proof).and_return(residential_address_proof)
-          allow(residential_address_proof).to receive(:success?).and_return(true)
+      context 'residential address and id address are different' do
+        let(:residential_address_proof) do
+          instance_double(Proofing::Resolution::Result)
+        end
+        let(:resolution_result) do
+          instance_double(Proofing::Resolution::Result)
+        end
+        let(:ipp_enrollment_in_progress) { true }
+        let(:applicant_pii) do
```

Comment:
i think we can use this mock applicant here: `MOCK_IDV_APPLICANT_STATE_ID_ADDRESS`

Reply:
Ooh. I feel like it would read nicer if that was possible. Though I think this was just a moved test case so I don't want to advocate for a rewrite.